programs/fungible-entangler/src/instructions/initialize_fungible_child_entangler_v0.rs

```diff
@@ -0,0 +1,83 @@
+use crate::state::*;
+use anchor_lang::prelude::*;
+use anchor_spl::token::{Mint, Token, TokenAccount};
+
+pub const CHILD_ENTANGLER_SIZE: usize = 8 + // key
+32 + // authority
+32 + // parent entangler
+32 + // mint
+32 + // storage
+8 + // go live
+8 + // freeze swap
+8 + // created
+1 + // bump
+1 + // storage bump
+200; // padding
+
+#[derive(AnchorSerialize, AnchorDeserialize, Clone, Default)]
+pub struct InitializeFungibleChildEntanglerV0Args {
+  pub authority: Option<Pubkey>,
+  pub go_live_unix_time: i64,
+  pub freeze_swap_unix_time: Option<i64>,
+}
+
+#[derive(Accounts)]
+#[instruction(args: InitializeFungibleChildEntanglerV0Args)]
+pub struct InitializeFungibleChildEntanglerV0<'info> {
```

Comment:
Authority of the parent entangler should be the only one that can create a child